elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -25,6 +25,8 @@ protected function load_dependencies() {
 		}
 		// Load ElasticPress
 		require_once __DIR__ . '/../../elasticpress/elasticpress.php';
+		// Load ElasticPress Debug Bar
+		require_once __DIR__ . '/../../debug-bar-elasticpress/debug-bar-elasticpress.php';
```

Comment:
Minor: What do you think about loading this from `elasticsearch/elasticsearch.php`?

My thinking is that this class more related to the implementation details of our ES integration and loading a separate plugin doesn't really fall into that.

Reply:
Then I would load both EP + Debug Bar from there.
I have added the loading of Debug Bar here because we were already loading EP here, but perhaps it may be better moving both of them?

Reply:
I don't see the advantage to moving the loading of dependencies. Right now we have the dependency loading nicely encapsulated inside `load_dependencies()` on the class, rather than scattered around inside elasticsearch.php. I would very much prefer keeping dependency loading as it is.

Reply:
I guess it depends on how you define a dependency. `debug-bar-elasticpress` doesn't really seem like a dependency for anything in `class-elasticsearch` (or any of our other ES related code).

Reply:
> I guess it depends on how you define a dependency. debug-bar-elasticpress doesn't really seem like a dependency for anything in class-elasticsearch (or any of our other ES related code).

This is a fair point. debug-bar-elasticpress looks more like a module to me, rather than a actual dependency, because VIP search does not _depend_ on it.
Should we then add a `load_modules()` function or something similar? Are there examples we can follow from existing plugins?